Docs/sphinx/Utility.rst

```diff
@@ -91,11 +103,45 @@ provided below. ::
   turbinflow.high.time_offset    = 0.0006
   turbinflow.high.verbose        = 2
 
+Subtleties
+~~~~~~~~~~
+To facilitate application of inflows generated from precursor Pele simulations, the data in the inflow files is now always interpreted as
+cell-centered, which is a departure from legacy implementations designed to use fully-periodic data originating from solvers with nodal data.
+However, some aspects of the implementation are still influenced by the legacy implementation. A main consideration in the current implementation
+is that an inflow generated from a precursor simulation (using either of the ``periodic_plt`` or ``diag_frame_plane`` generation options) can
+be directly applied as an inflow to a second simulation on the same grid without interpolation or data shifting.
+
+Each TurbInflow plane contains a valid box, plus one ghost cell on the low side and two ghost cells on the high side. But the dimension listed
```

Comment:
But the dimension listed in the ``HDR`` corresponds ... -> The dimensions listed in the ``HDR`` correspond to ...